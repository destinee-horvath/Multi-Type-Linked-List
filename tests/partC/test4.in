NEW 2
{0}
idk
NEW 3
i
need
ideas
NEW 4
some
other
{0}
list
VIEW ALL
NEW 5
another
really
weird
list
{1}
VIEW-NESTED 2
NEW 4
super
nested
{2}
end
VIEW-NESTED 3