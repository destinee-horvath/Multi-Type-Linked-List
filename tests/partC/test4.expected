INVALID COMMAND: NEW
INVALID COMMAND: INPUT
List 0: i -> need -> ideas
Nested 1: some -> other -> {List 0} -> list
Number of lists: 2
List 0
Nested 1
Nested 2: another -> really -> weird -> list -> {List 1}
another -> really -> weird -> list -> {some -> other -> {i -> need -> ideas} -> list}
Nested 3: super -> nested -> {List 2} -> end
super -> nested -> {another -> really -> weird -> list -> {some -> other -> {i -> need -> ideas} -> list}} -> end
